### PR TITLE
style(webapp): styles refactor

### LINKED ISSUE #247: styles refactor

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #247 

### Steps to test

1. Run the project and open localhost:3000
2. see the improved styles

#### CheckList

- [X] Follow the proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/components/PieChartReport/styles.js [hunk 1]
```diff
@@ -1,47 +1,26 @@
...
-  },
-  titleContainer: {
-    display: 'flex'
-  },
-  titleContainerExpense: {
-    display: 'flex',
-    width: '90%',
-    height: '65px',
-    borderBottom: '2px solid rgba(0, 0, 0, 0.12)'
+    margin: '16px 0 16px 0'
```
Comment (AngeloCG97) [inline webapp/src/components/PieChartReport/styles.js:3]: Use theme.spacing

File: webapp/src/components/DelegateDetails/styles.js [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
   verticalLine: {
-    display: 'flex',
-    position: 'relative',
-    borderLeft: '2px solid rgba(0, 0, 0, 0.12)',
-    height: '250',
-    width: '2px',
-    right: '30px'
+    borderLeft: '1px solid rgba(225,225,225,.99)',
+    borderBottom: '1px solid rgba(225,225,225,.99)',
+    marginBottom: '8px'
```
Comment (AngeloCG97) [inline webapp/src/components/DelegateDetails/styles.js:5]: Use theme.spacing

File: webapp/src/components/BarChartReport/styles.js [hunk 3]
```diff
@@ -1,53 +1,48 @@
...
-    fontSize: '18px',
-    fontWeight: 500,
-    lineHeight: 1.56,
-    letterSpacing: '-0.4px'
+  legentCircle: {
+    width: 12,
+    height: 12,
+    marginLeft: '4vw',
+    marginTop: 3,
+    marginRight: 8,
```
Comment (AngeloCG97) [inline webapp/src/components/BarChartReport/styles.js:45]: use theme.spacing

File: webapp/src/components/BarChartReport/styles.js [hunk 4]
```diff
@@ -1,53 +1,48 @@
...
   chartContainer: {
-    width: '100%',
-    display: 'flex',
-    alignItems: 'flex-start',
-    flexDirection: 'column',
-    marginTop: 24,
-    '& #chart-scroll-id': {
-      minWidth: '925px',
-      width: '100%'
+    margin: '16px 0 16px 0'
```
Comment (AngeloCG97) [inline webapp/src/components/BarChartReport/styles.js:3]: Use theme.spacing

File: webapp/src/components/StackedBarChartReport/styles.js [hunk 5]
```diff
@@ -1,43 +1,48 @@
 export default theme => ({
   chartContainer: {
-    width: '50%',
+    margin: '16px 0 16px 0'
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/styles.js:3]: Use theme.spacing

File: webapp/src/components/StackedBarChartReport/styles.js [hunk 6]
```diff
@@ -1,43 +1,48 @@
...
-    display: 'flex',
-    position: 'relative',
-    left: '10%',
-    alignItems: 'center'
+  legentCircle: {
+    width: 12,
+    height: 12,
+    marginLeft: '2vw',
+    marginTop: 3,
+    marginRight: 8,
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/styles.js:45]: Use theme.spacing

File: webapp/src/routes/DelegateReport/styles.js [hunk 7]
```diff
@@ -1,55 +1,44 @@
...
     marginTop: '20vh'
   },
   headPage: {
-    display: 'flex'
+    display: 'flex',
+    justifyContent: 'space-between',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      '& #treasury-container-id': {
+        marginTop: '8px'
```
Comment (AngeloCG97) [inline webapp/src/routes/DelegateReport/styles.js:15]: Use theme.spacing

File: webapp/src/routes/DelegateReport/styles.js [hunk 8]
```diff
@@ -1,55 +1,44 @@
...
-      display: 'flex',
-      marginLeft: '12px',
-      justifyContent: 'end',
-      alignItems: 'center'
+    marginBottom: '24px',
+    display: 'flex',
+    justifyContent: 'end',
+    alignItems: 'center',
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 8,
```
Comment (AngeloCG97) [inline webapp/src/routes/DelegateReport/styles.js:36]: Use theme.spacing

File: webapp/src/routes/DelegateReport/styles.js [hunk 9]
```diff
@@ -1,55 +1,44 @@
...
+    }
   },
   filtersContainer: {
     marginTop: '24px',
-    '& #id-radio-election-container': {
-      display: 'flex',
-      marginLeft: '12px',
-      justifyContent: 'end',
-      alignItems: 'center'
+    marginBottom: '24px',
```
Comment (AngeloCG97) [inline webapp/src/routes/DelegateReport/styles.js:31]: Use theme.spacing

File: webapp/src/routes/DelegateReport/styles.js [hunk 10]
```diff
@@ -1,55 +1,44 @@
...
-    position: 'relative'
+    '& span': {
+      fontSize: '18px',
+      fontWeight: 500,
+      lineHeight: 1.56,
+      letterSpacing: '-0.4px'
+    }
   },
   filtersContainer: {
     marginTop: '24px',
```
Comment (AngeloCG97) [inline webapp/src/routes/DelegateReport/styles.js:30]: Use theme.spacing

File: webapp/src/routes/ExpenseReport/styles.js [hunk 11]
```diff
@@ -1,7 +1,6 @@
 export default theme => ({
   root: {
-    marginRight: '8px',
-    marginLeft: '8px',
+    margin: '8px',
```
Comment (AngeloCG97) [inline webapp/src/routes/ExpenseReport/styles.js:3]: Use theme.spacing

File: webapp/src/routes/ExpenseReport/styles.js [hunk 12]
```diff
@@ -11,88 +10,44 @@ export default theme => ({
...
-      letterSpacing: '-0.4px',
-      [theme.breakpoints.down('550')]: {
-        marginTop: '16px'
-      }
+      letterSpacing: '-0.4px'
     }
   },
   filtersContainer: {
-    marginTop: '24px',
+    marginTop: '16px',
```
Comment (AngeloCG97) [inline webapp/src/routes/ExpenseReport/styles.js:21]: Use theme.spacing

File: webapp/src/routes/ExpenseReport/styles.js [hunk 13]
```diff
@@ -11,88 +10,44 @@ export default theme => ({
...
-    alignItems: 'center'
+  pieChartContainer: {
+    width: '35vw',
+    [theme.breakpoints.down('md')]: {
+      width: '100%'
+    }
   },
   tableContainer: {
-    marginTop: '80px',
+    marginTop: '24px',
```
Comment (AngeloCG97) [inline webapp/src/routes/ExpenseReport/styles.js:50]: Use theme.spacing

File: webapp/src/routes/ExpenseReport/styles.js [hunk 14]
```diff
@@ -131,11 +69,16 @@ export default theme => ({
...
     }
   },
   disableLink: {
     pointerEvents: 'none',
     cursor: 'pointer'
+  },
+  dividerLine: {
+    borderLeft: '1px solid rgba(225,225,225,.99)',
+    borderBottom: '1px solid rgba(225,225,225,.99)',
+    marginBottom: '8px'
```
Comment (AngeloCG97) [inline webapp/src/routes/ExpenseReport/styles.js:82]: Use theme.spacing

File: webapp/src/routes/IncomeReport/styles.js [hunk 15]
```diff
@@ -1,7 +1,6 @@
 export default theme => ({
   root: {
-    marginRight: '8px',
-    marginLeft: '8px',
+    margin: '8px',
```
Comment (AngeloCG97) [inline webapp/src/routes/IncomeReport/styles.js:3]: Use theme.spacing

File: webapp/src/routes/IncomeReport/styles.js [hunk 16]
```diff
@@ -11,75 +10,28 @@ export default theme => ({
...
-      letterSpacing: '-0.4px',
-      [theme.breakpoints.down('550')]: {
-        marginTop: '16px'
-      }
+      letterSpacing: '-0.4px'
     }
   },
   filtersContainer: {
-    marginTop: '30px',
+    marginTop: '16px',
```
Comment (AngeloCG97) [inline webapp/src/routes/IncomeReport/styles.js:21]: Use theme.spacing

File: webapp/src/routes/IncomeReport/styles.js [hunk 17]
```diff
@@ -11,75 +10,28 @@ export default theme => ({
...
     '& #id-radio-election-container': {
       display: 'flex',
       marginLeft: '12px',
```
Comment (AngeloCG97) [inline webapp/src/routes/IncomeReport/styles.js:24]: Use theme.spacing

File: webapp/src/routes/IncomeReport/styles.js [hunk 18]
```diff
@@ -11,75 +10,28 @@ export default theme => ({
...
-    width: '4px',
-    height: '31px',
-    backgroundColor: '#00c2bf'
-  },
-  titleContainer: {
-    display: 'flex',
-    alignItems: 'center'
-  },
   tableContainer: {
     marginTop: '24px',
```
Comment (AngeloCG97) [inline webapp/src/routes/IncomeReport/styles.js:36]: Use theme.spacing

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]
LGTM

**Pull request merged**